production.ini

```diff
@@ -66,10 +66,14 @@ use = egg:bodhi-server
 # wiki_url = https://fedoraproject.org/w/api.php
 # test_case_base_url = https://fedoraproject.org/wiki/
 
-# URL of the resultsdb for integrating checks and stuff
-# resultsdb_api_url = https://taskotron.fedoraproject.org/resultsdb_api/
```

Comment:
This gets used to display the test results in the UI. I think we need to keep it for now.